addons/mail/models/mail_thread.py

```diff
@@ -1653,7 +1653,9 @@ def _message_get_suggested_recipients(self):
         """ Returns suggested recipients for ids. Those are a list of
         tuple (partner_id, partner_name, reason), to be managed by Chatter. """
         result = dict((res_id, []) for res_id in self.ids)
-        if 'user_id' in self._fields:
+        if 'user_id' in self._fields and \
+        self._fields['user_id'].type == 'many2one' and \
+        self._fields['user_id'].comodel_name == 'res.users':
```

Comment:
Maybe we could use a variable like
```suggestion
        user_field = self._fields.get('user_id')
        if user_field and user_field.type == 'many2one' and user_field.comodel_name == 'res.users':
```

Reply:
Yes, i have updated thanks